docs/src/correcting_class_imbalance.md

```diff
@@ -0,0 +1,23 @@
+# Correcting Class Imbalance
+
+## Oversampling and undersampling methods
+
+Models providing oversampling or undersampling methods, to correct for class imbalance,
+are listed under [Class Imbalance](@ref). In particular, several popular algorithms are
+provided by the [Imbalance.jl]() package, which includes detailed documentation and
+tutorials.
+
+## Incorporating class imbalance in supervised learning pipelines
+
+One or more oversampling/undersampling algorithms can be fused with an MLJ classifier
+using the [`BalancedModel`](@ref) wrapper. This creates a new classifier which can be
+treated like any other; resampling to correct for class imbalance, relevant only for
+*training* of the atomic classifier, is then carried out internally. If, for example, one
+applies cross-validation to the wrapped classifier (using [`evaluate!`](@ref), say) then
+this means over/undersampling is then repeated for each training fold automatically.
+
+Refer to the [MLJBalancing.jl]() documentation for further details.
```

Comment:
What are the plans for the link here? If nothing in particular then I may suggest the [MLJBalancing page](https://juliaai.github.io/Imbalance.jl/dev/algorithms/mlj_balancing/) found in `Imbalance.jl` docs. I will likely also rachet up the formality in the `BalancedModel` section using the recent docstring.

Reply:
Okay. I didn't notice you included the `BalancedModel` docstring here. I think this should be more than enough for `BalancedModel` you may not link anywhere for "MLJBalancing documentation". `BalancedBaggingClassifier` also has its own nice docstring I believe.

Reply:
Oops. Fixed as suggested.